mm2src/coins/eth.rs

```diff
@@ -1152,7 +1176,7 @@ impl MarketCoinOps for EthCoin {
             .ok_or(VerificationError::PrefixNotFound)?;
         let address = self
             .address_from_str(address)
-            .map_err(VerificationError::AddressDecodingError)?;
+            .mm_err(|err| VerificationError::AddressDecodingError(err.to_string()))?;
```

Comment:
I think we should split `MmAddressError` into `MyAddressError` and `ParseAddressError` (or something similar). The last error even could be individual for UTXO, ETH and other coins.

Reply:
done